examples/soak/setup_ecs_cluster.yml

```diff
@@ -0,0 +1,242 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: This stack starts an example load balanced Fargate task.
+
+Parameters:
+  VpcStackName:
+    Type: String
+    Default: soak-vpc
+    Description: The name of the parent Fargate networking stack that you created. Necessary
+                 to locate and reference resources created by that stack.
+
+Resources:
+  # ECS Resources
+  ECSCluster:
+    Type: AWS::ECS::Cluster
+    Properties:
+      ClusterName: 'crux-soak'
+      ClusterSettings:
+        - Name: 'containerInsights'
+          Value: 'enabled'
+
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: 'crux-soak'
+      RetentionInDays: 365
+
+  ECRRepository:
+    Type: AWS::ECR::Repository
+    Properties:
+      RepositoryName: 'crux-soak'
+
+  # A security group for the containers we will run in Fargate.
+  # Three rules, allowing network traffic from a public facing load
+  # balancer, a private internal load balancer, and from other members
+  # of the security group.
+  #
+  # Remove any of the following ingress rules that are not needed.
+  FargateContainerSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Access to the Fargate containers
+      VpcId:
+        Fn::ImportValue:
+          !Join [':', [!Ref 'VpcStackName', 'VPCId']]
+
+  # Load balancers for getting traffic to containers.
```

Comment:
I don't mind comments sticking about (I normally wouldn't have comments in CF templates explaining what the AWS services are, assumption is that people will look these up in AWS docs if need be) but let's make sure the comments that do stick about still apply (e.g. we're not using a private load balancer)